README.md

```diff
@@ -188,6 +188,8 @@ Validators
 ----------
 Here are all the validators Yamale knows about. Every validator takes a `required` keyword telling Yamale whether or not that node must exist. By default every node is required. Example: `str(required=False)`
 
+You can require an optional node to have a value. A value is not needed by default. In this example, an optional node must have a value if the node is used: `str(required=False, none=False)`
+
```

Comment:
Maybe:
```
You can also require that an optional value is not `None` by using the `none` keyword. By default Yamale will accept `None` as a valid value for a key that's not required. Reject `None` values with `none=False` in any validator. Example: `str(required=False, none=False)`.
```

Reply:
@blopker, I've updated the README with the new text. I also updated the version from 1.7.1 to 1.7.2. If you want it to stay at 1.7.1, let me know.

Reply:
Generally I'll handle the versioning, but this is fine. Thanks!